src/Raven.Server/Documents/Handlers/Processors/OngoingTasks/OngoingTasksHandlerProcessorForGetConnectionString.cs

```diff
@@ -108,57 +28,28 @@ public override async ValueTask ExecuteAsync()
                 return;
 
             var connectionStringName = RequestHandler.GetStringQueryString("connectionStringName", false);
-            var type = RequestHandler.GetStringQueryString("type", false);
+            var typeString = RequestHandler.GetStringQueryString("type", false);
+            if (Enum.TryParse(typeString, ignoreCase: true, out ConnectionStringType connectionStringType) == false && typeString != null)
+                throw new NotSupportedException($"Unknown connection string type: {connectionStringType}");
```

Comment:
IIRC we are usually throwing BadRequestException + invert checks in if,  if typestring is null, no need to do parsing

Reply:
Done.